src/test/resources/alma/HT016709661.xml

```diff
@@ -164,15 +164,39 @@
     <subfield code="n">HBZ Network</subfield>
   </datafield>
   <datafield tag="MNG" ind1=" " ind2=" ">
-    <subfield code="f">ILS</subfield>
+    <subfield code="c">System</subfield>
+    <subfield code="f">HBZ</subfield>
     <subfield code="i">marc21</subfield>
+    <subfield code="h">20141122133000.0</subfield>
     <subfield code="k">01</subfield>
     <subfield code="e">false</subfield>
+    <subfield code="d">2021-01-05 21:45:48 Europe/Berlin</subfield>
     <subfield code="g">018412741-HBZ01</subfield>
     <subfield code="j">90</subfield>
     <subfield code="a">import</subfield>
     <subfield code="b">2020-11-23 03:27:51 Europe/Berlin</subfield>
   </datafield>
+  <datafield tag="POR" ind1=" " ind2=" ">
+    <subfield code="x">System</subfield>
+    <subfield code="a">53482457280006441</subfield>
+    <subfield code="M">49HBZ_NETWORK</subfield>
+    <subfield code="p">61480999250006441</subfield>
+    <subfield code="q">Lizenzfreie E-Ressourcen der ZDB, die nicht &#252;ber die EZB verwaltet werden (lzzdb ohne URL der UR)</subfield>
+    <subfield code="c">static</subfield>
+    <subfield code="A">49HBZ_DUE</subfield>
```

Comment:
Here you see indicating Düsseldrof by "<subfield code="A">49HBZ_DUE</subfield>"  while mapped to DE-605 by taking  <subfield code="a">53482457280006441</subfield> (see 5 lines up) into account.
What does the POR__.A means anyway?

Reply:
https://github.com/hbz/lobid-resources/issues/1135#issuecomment-787710851

While "M" is the Member Code, "A" is a refrence for "Available for Institution". I don't know if this relevant for us.